### PR TITLE
remove prettier

### PR DESCRIPTION
**Pull request by sachinraja**

### CODE REVIEW

File: e2e/nextjs-routes.d.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
 // THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.'
 // Run `yarn nextjs-routes` to regenerate this file.
 /* eslint-disable */
+// prettier-ignore
```
Comment (tatethurston) [inline e2e/nextjs-routes.d.ts:4]: Does this ignore the next node and not the whole file? https://prettier.io/docs/en/ignore.html#javascript

Comment (tatethurston) [reply e2e/nextjs-routes.d.ts:4]: `"nextjs-routes"` is the next node and the only dynamic piece that is likely to be formatted, just curious if there are semantics for a whole file ignore that I'm unaware of

Comment (sachinraja) [reply e2e/nextjs-routes.d.ts:4]: Oh you're right, that's unfortunate.

Comment (sachinraja) [reply e2e/nextjs-routes.d.ts:4]: Guess we just force users to ignore it?

Comment (tatethurston) [reply e2e/nextjs-routes.d.ts:4]: I wonder if [range ignore](https://prettier.io/docs/en/ignore.html#range-ignore) works in JS?

Comment (sachinraja) [reply e2e/nextjs-routes.d.ts:4]: Unfortunately not https://github.com/prettier/prettier/issues/5287

Comment (tatethurston) [reply e2e/nextjs-routes.d.ts:4]: That's unfortunate. We only have 3 nodes so I suppose we could add the ignore above each module declaration?

Comment (sachinraja) [reply e2e/nextjs-routes.d.ts:4]: done https://github.com/tatethurston/nextjs-routes/pull/27

**Pull request merged**